test/graphql/types/Organization/organizationsConnection.test.ts

```
@@ -0,0 +1,222 @@
+		]);
+	});
+
+	it("should return organizations when valid arguments are provided", async () => {
+		const args = {
+			first: 10,
+			after: null,
+			last: null,
+			before: null,
+		};
+
+		const result = await resolve(null, args, mockContext as any);
```

Biome (1.9.4)

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)